src/main/java/frc/robot/subsystems/pivot/Pivot.java

```diff
@@ -25,16 +25,20 @@ public void periodic() {
     // This method will be called once per scheduler run
   }
 
-  public void setPivotFromSpeakerDistance(double speakerDistance) {
-    io.setPivotFromSpeakerDistance(speakerDistance);
+  public void setPivotAngle(double angle) {
```

Comment:
Add docstrings for these method

Reply:
Also @Ishan1522 , what's the deal with io? Maybe you can explain it to me when I come to a meeting lol

Reply:
💀💀💀💀💀💀